helm/alfresco-content-services/charts/alfresco-sync-service/Chart.yaml

```diff
@@ -9,4 +8,4 @@ keywords:
 maintainers:
 - name: Alfresco Apps Repo Team
 name: alfresco-sync-service
-version: 3.0.6
+version: 3.0.7
```

Comment:
Is this related to search?

Reply:
Fair point, no it's not, I will revert from this PR.